vault/changelog.release.2021-11-23.md

```diff
@@ -0,0 +1,69 @@
+---
+id: Ku1qjbDYlsxk4rfMoFEza
+title: "0.70"
+desc: ''
+updated: 1637691256859
+created: 1637687416614
+---
+
+Dendron 0.70 has sprouted  🌱
+
+Dendronites can now use `Dendron: Initialize Workspace` to create [[native workspaces|dendron.topic.workspace.native]], variable substitution is possible in the desc note frontmatter, and improvements have been made to task note decorators (the task info that magically renders whenever a note link is pointing to a task note). Jump in and take a look!
```

Comment:
1. Most people don't know these features are (native workspaces, variable substitution, and task notes). For each of these items, I would make sure to link to the source docs (like you did with native workspaces).
1. Variable substitution with `desc` is great but shouldn't be the highlight of this week. 
1. Task note changes have been reverted. I would spend the highlight line for this week highlighting native workspaces and dropping the idea that people can use it with their code